src/components/Layout.js

```diff
@@ -27,7 +27,6 @@ const Main = styled.main`
 const Layout = ({ children }) => {
   return (
     <>
-      <BackgroundLottie />
```

Comment:
Wasn't this used for the animated background? I see it still works in the deploy preview, just wondering what's going on :sweat_smile:

Reply:
Correct, well spotted! I should've given more detail on that straight away. 😅 

I decided to remove the animated background from the individual episode pages but keep it on the main homepage and the podcast homepage. I felt like, for the episode pages, there's too much stuff going on with the episode player, the boostagrams, the show notes _and_ the animated background. So I thought we could keep the animation on the more static main pages (main homepage, podcast homepage, eventual blog homepage) and remove it for the content heavy pages (episode pages, blog posts) to keep them more focused and distraction free.

To do that in a nice way so that the animation doesn't re-render when switching from the main homepage to the podcast homepage, these pages need to inject their layout (including the animation) into the main page (see [here](https://github.com/seetee-io/seetee-io/pull/30/files#diff-26ca35c3be6d07c84bb77f1e63a1b19487279d5b9c9a9392179b3a35647e1254R82)) while the main page's layout by default doesn't have the animation (See [here](https://github.com/seetee-io/seetee-io/pull/30/files#diff-7ed85bcba7790b36407e1a6dda8c455dcef4aae62ebe666a363fc582308e1120R18)).

This results in the animation continuously playing when switching from _Index_ ➡️ _Podcasts_  while not showing at all for the episode pages:

https://user-images.githubusercontent.com/10026790/146934789-524e390a-fea8-450a-92fe-543636ccdf9a.mov

Hope this somehow makes sense. If not let me know! 😄